README.md

```diff
@@ -8,20 +8,22 @@ Finally, the package allows deployment metadata to be ingested into the files fr
 
 See our [wiki page](https://github.com/aodn/imos-toolbox/wiki) page for more details.
 
-# Distribution/Requirements
+# Distribution
 
 The **stable** releases may be obtained [here](https://github.com/aodn/imos-toolbox/releases). The releases contain both the source code and binary applications (executables).
 
-We support Windows and Linux. The toolbox may be used as a **Matlab stand-alone library** or **stand-alone application** (**No Matlab is required**).
 
-For a **stand-alone library** usage, **Matlab R2018b+** is required ( since version 2.6.1).
+# Requirements
+
+We support Windows and Linux. The toolbox may be used as a **Matlab stand-alone library** or **stand-alone application** (**No Matlab is required**).
 
-# Install
+For a **stand-alone library** usage, **Matlab R2018b** and the Matlab Component Runtime (v95) or newer is required ( since version 2.6.1).
```

Comment:
You don't need the MCR for use with Matlab.

Reply:
oops - done